README.md

```diff
@@ -47,6 +42,4 @@ It assumes you have the Travis CLI installed and you are already logged into Tra
 
 It is worth noting that it is possible to run the PR script locally by setting the environment variables. For example:
 
- `TRAVIS=true TRAVIS_REPO_SLUG=nhsuk/connecting-to-services TRAVIS_PULL_REQUEST=<PR number of an image which still exists>  RANCHER_TEMPLATE_NAME=c2s-pharmacy-finder SPLUNK_HEC_TOKEN=<splunk token> GITHUB_ACCESS_TOKEN=<personal access token> RANCHER_ENVIRONMENT=<rancher env name> ./rancher_pr_deploy.sh`
-
- If testing changes to the rancher template during development  RANCHER_TEMPLATE_BRANCH_NAME can also be set (see above for details).
+ `TRAVIS=true TRAVIS_REPO_SLUG=nhsuk/connecting-to-services TRAVIS_PULL_REQUEST=<PR number of an image which still exists>  GITHUB_ACCESS_TOKEN=<personal access token> RANCHER_ENVIRONMENT=<rancher env name> ./deploy.sh`
```

Comment:
I'm not able to run the script locally from the root directory for this repo as I used to be able to due to the hard coded paths to the common scripts (e.g. ./scripts/ci-deployment/travis/build-docker-images.sh). Not sure what you can do about this.